src/Components/__stories__/TooltipError.story.tsx

```diff
@@ -0,0 +1,108 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+
+import styled from "styled-components"
+import Button from "../Buttons/Default"
+import Input from "../Input"
+import { Popover } from "../Popover"
+
+const Container = styled.div`
+  padding: 50px;
+`
+
+class ErrorShower extends React.Component<any, any> {
+  constructor(props) {
+    super(props)
+    this.state = { count: 0, errorMessage: "" }
+  }
+
+  showTip = () => {
+    const count = this.state.count
+    this.setState({
+      errorMessage: this.props.message + ` (${count})`,
+      count: count + 1,
+    })
```

Comment:
Does this mean the error message and the count state are now out of sync? Should it be:

```javascript
const count = this.state.count + 1

this.setState({
  errorMessage: this.props.message + ` (${count})`,
  count
})
```

Reply:
I didn't think about this that much- it was my first story and I needed something to make sure the text was changing. I'll think about this.